tools/proofers/mktable.php

```diff
@@ -43,8 +43,8 @@
 <body>
 <form method="POST">
 <input type="submit" value="Create">
-<input type="text" name="row" value="<?php echo $row; ?>" size="2"> rows and
-<input type="text" name="col" value="<?php echo $col; ?>" size="2"> columns table
+<input type="number" name="row" value="<?php echo $row; ?>" style='width: 3em;'> rows and
+<input type="number" name="col" value="<?php echo $col; ?>" style='width: 3em;'> columns table
```

Comment:
`min=1` and `required` for both would make sense. 

Handles `5.0` and `5e0` well.

Reply:
Added `min`.